src/Components/SavedReadings/SavedReadings.js

```diff
@@ -1,10 +1,70 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { cardImages } from '../../cardImageData'
 import './SavedReadings.css'
 
-function SavedReadings() {
-  return (
-    <h1>SavedReadings</h1>
-  );
+class SavedReadings extends Component {
+  constructor({ userSavedReadings }) {
+    super()
```

Comment:
Please correct me if I'm wrong because I'm not confident in this but isn't it best practice to pass the super the props as well? I feel like I heard that in class before but I'm not sure. Love that you are destructing when you pass props in to begin with though! Very clean, love that and should implement some of that myself!!